docs/core/option.md

```diff
@@ -111,4 +203,40 @@ IReadOnlyList<int> list = [2, 4, 6];
 Option<IReadOnlyList<int>> listOfOnlyEvenNumbers =
     list.Traverse(x => x % 2 == 0 ? Some(x) : None);
 // Some([2, 4, 6])
+```
+
+### Sequence
+
+Use `Sequence` to traverse without the mapping step.
+
+```csharp
+IReadOnlyList<Option<int>> list = [Some(2), Some(4), Some(6)];
```

Comment:
Let's add a `None` in there too for illustrative purposes

```suggestion
IReadOnlyList<Option<int>> list = [Some(2), Some(4), None, Some(6)];
```

Reply:
Done